src/exchange/account/delegation/account-delegation.html

```diff
@@ -23,6 +23,15 @@
                 <div class="form-group">
                     <label class="sr-only" for="delegationSearch"
                            data-translate="common_search"></label>
+                    <div class="oui-select oui-select_inline">
```

Comment:
Wouldn't a `oui-select` work ?

Reply:
Looks like it's not possible or too complex to get it done. I need to add a custom option to select all domains. On selecting it, all domains are shown.

Reply:
https://ovh-ux.github.io/ovh-ui-kit-documentation/#!/oui-angular/select

Wouldn't a `onChange` work ?  If you select `All domains`, then you display all domains, else you display only the selected domain

Reply:
I replaced select with oui-select